docs/UserGuide/System Tools/NodeTool.md

```diff
@@ -34,7 +37,10 @@ you can specify the IP address and port of the cluster during startup.
 IP is the IP (or hostname) of the node that you expect to connect to,
 and port is the JMX service port specified when the IoTDB cluster is started.
 
-The default values are 127.0.0.1 and 31999, respectively.
+The default values are `127.0.0.1` and `31999`, respectively.
+
+When JMX authentication service is enabled, the user name and password of JMX service should be specified. 
```

Comment:
These two lines should be put after the next two lines? And it's better to point out how to change authentication parameters, such as -u,-pw.

Reply:
sure, thanks